hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/blockmanagement/TestAvailableSpaceRackFaultTolerantBPP.java

```diff
@@ -189,6 +189,13 @@ public void testChooseDataNode() {
     }
   }
 
+  @Test
+  public void testDefaultConfigValue() {
+    Assert.assertEquals(
+            conf.get("DFS_NAMENODE_AVAILABLE_SPACE_RACK_FAULT_TOLERANT_BLOCK_PLACEMENT_POLICY_BALANCED_SPACE_PREFERENCE_FRACTION_KEY")
+            ,0.6f);
+  }
+
```

Comment:
This isn't required, this isn't checking the BPP, but how the conf object works. you can remove this test.

Reply:
> This isn't required, this isn't checking the BPP, but how the conf object works. you can remove this test.

Thanks @ayushtkn for your review. I have removed the test case, but not sure if the CI will pass if no tests added.  hope it works.